eslint/babel-eslint-parser/src/convert/convertTokens.cjs

```diff
@@ -31,7 +31,7 @@ function convertTemplateType(tokens, tl) {
     templateTokens = [];
   }
 
-  tokens.forEach(token => {
+  (tokens || []).forEach(token => {
```

Comment:
When can `tokens` be an empty array?

Reply:
Same as https://github.com/babel/babel/pull/13338#discussion_r635442417

Reply:
Removed in favour of https://github.com/babel/babel/pull/13338/commits/3769db3a70de3243b2bdcdd71f550a8074c03519